libs/docs-components/src/code-preview/component/code-preview.component.html

```diff
@@ -0,0 +1,9 @@
+@if (example) {
+  <div class="daff-docs-code-preview__content" *ngComponentOutlet="example.component"></div>
```

Comment:
`<div class="daff-docs-code-preview__content">` this div does not show up in the DOM.

<img width="290" alt="Screen Shot 2024-07-15 at 8 51 14 PM" src="https://github.com/user-attachments/assets/df55f986-0f57-4725-82c2-d0c16184513d">

Reply:
fixed